guides/common/modules/proc_remediating-compliance-failures.adoc

```diff
@@ -0,0 +1,33 @@
+[id="remediating-compliance-failures_{context}"]
+= Remediating compliance failures
+
+You can examine compliance reports and, in some cases, remediate cases of non-compliance.
```

Comment:
what does "in some cases" depend on? Your imported OpenSCAP content/rules? the severity? If the vender (aka RH for RHEL) provides an automated remediation workflow?

Reply:
I'd like to clarify that too in another PR that will add information about the new remediation wizard. Because the wizard is also available only in "some cases". For now, we can keep this as it is because this is what the docs say already.